### DISCUSSION TITLE
Is there a way to move pages?

### DISCUSSION THREAD

**Discussion by Balky79** [redimp/otterwiki#139] [Q&A] [answered]
I've just learned here that changing URL is not possible, but I can't find anywhere if there is a way to "move" pages that are in some bucket?

**Comment by redimp** [comment, marked as answer]
Hey @Balky79,

I'm not sure what you mean with "bucket". An Otter Wiki support [Subdirectories](https://otterwiki.com/-/help#subdirectories). You can move pages by renaming them, e.g. with

```
Home
Pool/PageA
PageB
```

and renaming `PageB` to `Pool/PageB` you get

```
Home
Pool/PageA
Pool/PageB
```

Does this answer your question?

**Reply by Balky79** [reply]
Sorry :D 

I don't know where my brain was yesterday, and I didn't even think of trying that... 

Yeah, it does help, and even renaming main page from Pool to 'Big Pool' works...  

Thanks :)
